### ISSUE TITLE
Request: Remember cursor/scroll position per file

### ISSUE THREAD

**Issue by Kuroonehalf** [focus-editor/focus#226]
Also brought up in the Discussions post I made recently, I wanted to make a bespoke issue for this feature. I don't know that I have to make much of an argument for its inclusion, since it has clear productivity benefits, but this is an indispensable feature for me in the code editors I've used. It's extra useful for me when writing dialog for my games, where files tend to get quite long, and being able to instantly pick up where I left off writing is a must have.

**Comment by focus-editor** [comment]
The position is remembered for up to 2 editors if you reopen them using any of the file dialogs. If you move back through cursor history then it scrolls to cursor right now - we'll make a proper editor history without that later.

If this issue is about remembering open editors between sessions, then that feature is also planned and was mostly implemented by right now disabled because of stability concerns temporarily

**Comment by Kuroonehalf** [comment]
Yeah, I meant open between sessions. So even if I pick up a file from last week, it takes me exactly to where I was working.
